### DISCUSSION TITLE
Failed to create SSL Socket

### DISCUSSION THREAD

**Discussion by technocp** [Xpra-org/xpra#3906] [Q&A] [answered]
Hi All,

I have been using xpra for remote access of my application over https with xpra-html5

previously I used to have public IP directly set on my linux vmware client and my users used to access it from their client browsers using my subdomain address

But in my new datacenter they do not provide public IP to be set directly on my linux vmware client, rather they keep it in thier firewall and then redirect all requests to my linux vmware client.

now the problem is that when i run the same old setup it gives me failed to create ssl socket in my xpra logs.

can someone please help and throw some light over making changes in my linux system configuration or xpra configuration to be able to successfully create ssl socket and access my application in xpra.

**Comment by totaam** [comment, marked as answer]
There aren't enough details here to begin to help you. You can find some generic guidelines here:
https://github.com/Xpra-org/xpra/wiki/Reporting-Bugs
Full command lines, versions, logs, exact error messages, `openssl` diagnostics, etc.
